app/move/fields/additional-information-base.js

```diff
@@ -0,0 +1,11 @@
+const additionalInformationBase = {
```

Comment:
Could use the existing pattern we've used for shared fields: `common.additional-information.js`?